src/amp-story-player/amp-story-player-impl.js

```diff
@@ -563,6 +563,7 @@ export class AmpStoryPlayer {
     this.maybeGetCacheUrl_(story.href).then((url) => {
       const {href} = this.getEncodedLocation_(url, visibilityState);
       iframe.setAttribute('src', href);
+      iframe.setAttribute('title', story.textContent.trim());
```

Comment:
Isn't this going to cause a tooltip?

Reply:
Not from my super quick testing, but I didn't try on the actual player: https://codepen.io/gmajoulet/pen/mdVLOVM

Reply:
I tried the player on desktop and emulating touch and didn't see a tooltip